qute.jdt/com.redhat.qute.jdt/src/main/java/io/quarkus/runtime/util/StringUtil.java

```diff
@@ -0,0 +1,238 @@
+package io.quarkus.runtime.util;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Locale;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
+/**
+ * This class is acopy/paste from https://github.com/quarkusio/quarkus/blob/main/core/runtime/src/main/java/io/quarkus/runtime/util/StringUtil.java

```

Comment:
is a copy of https://github.com/quarkusio/quarkus/blob/8cf05e7e6cf2ba15a41590561d8be14c2d5ce995/core/runtime/src/main/java/io/quarkus/runtime/util/StringUtil.java

(always prefer permalinks)

Reply:
fixed